composer.json

```diff
@@ -20,7 +20,7 @@
         }
     },
     "require": {
-        "php": "^7.0"
+        "php": "^7.4"
```

Comment:
This change is easy to make without using 7.4 features.

Reply:
Yes, but I was planning on releasing a 2.0 version, because #7 changes the behaviour, so why not require a more recent version of PHP?